### ISSUE TITLE
Add travis.ci integration

### ISSUE THREAD

**Issue by jwillemsen** [jpmeijers/RN2483-Arduino-Library#26]
Adding an issue to make this public, I am working on my fork on adding travis-ci integration using platform.io. Have some issues with platform.io and the sodaq boards, waiting on response of the platform.io team. When this works I will make a new PR to add this, it enables automatic testing of all examples, including pull requests. 

@jpmeijers Please assign this issue to me

**Comment by jwillemsen** [comment]
@jpmeijers After you merge #27 can you go to https://travis-ci.org/ and register for an account and enable travis-ci for this repository?

**Comment by jwillemsen** [comment]
Looks there is an issue with platformio.org and the sodaq boards, see https://community.platformio.org/t/missing-variant-h-and-sodaq-board/1325/3

**Comment by jwillemsen** [comment]
Found the code that causes the warning, made some pull requests to sodaq to get the warning resolved
